intel/yaml/intel-istio-sgx-gateway.yaml

```diff
@@ -1,237 +1,243 @@
-                value: tcsclusterissuer.tcs.intel.com
-              - name: EXTERNAL_CA
-                value: ISTIOD_RA_KUBERNETES_API
-              - name: PILOT_CERT_PROVIDER
-                value: k8s.io/tcsclusterissuer.tcs.intel.com/sgx-signer
-            volumes:
-            - emptyDir:
-              name: workload-socket
-            - emptyDir:
-              name: credential-socket
-            - emptyDir:
-              name: sgx-libraries
-            - emptyDir:
-              name: ctk-tokens
-    gateways:
-      istio-ingressgateway:
-        podAnnotations:
-          sgx.intel.com/quote-provider: aesmd
-        additionalContainers:
-          - name: testsds
-            image: sds-server/sds-server:latest
-            imagePullPolicy: IfNotPresent
-            command: ["bash", "-c", "sleep 5s; ls -la /home/istio-proxy/sgx/lib; ./sds/sds-server"]
-            lifecycle:
-              postStart:
-                exec:
-                  command: ["/bin/sh", "-c", "./home/istio-proxy/prepare.sh"]
-            resources:
-              limits:
-                sgx.intel.com/enclave: 1
-                sgx.intel.com/epc: 1Mi
-              requests:
-                sgx.intel.com/enclave: 1
-                sgx.intel.com/epc: 1Mi
-            env:
-            - name: POD_NAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.name
-            - name: POD_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
-            - name: SERVICE_ACCOUNT
-              valueFrom:
-                fieldRef:
-                  fieldPath: spec.serviceAccountName
-            - name: NEED_QUOTE
-              value: "false"
-            - name: RANDOM_NONCE
-              value: "false"
-            - name: CERT_SIGNER_DOMAIN
-              value: tcsclusterissuer.tcs.intel.com
-            - name: EXTERNAL_CA
-              value: ISTIOD_RA_KUBERNETES_API
-            - name: PILOT_CERT_PROVIDER
-              value: k8s.io/tcsclusterissuer.tcs.intel.com/sgx-signer
-            volumeMounts:
-            - name: workload-socket
-              mountPath: /var/run/secrets/workload-spiffe-uds
-            - name: credential-socket
-              mountPath: /var/run/secrets/credential-uds
-            - name: sgx-libraries
-              mountPath: /home/istio-proxy/sgx/lib
-            - name: podinfo
-              mountPath: /etc/istio/pod
-            - name: istio-token
-              mountPath: /var/run/secrets/tokens
-            - name: istiod-ca-cert
-              mountPath: /var/run/secrets/istio
-            - name: istio-data
-              mountPath: /var/lib/istio/data
-            - name: ctk-tokens
-              mountPath: /opt/intel/cryptoapitoolkit/tokens
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  name: istio
+spec:
+  profile: default
+  hub: "intel"
+  tag: "1.16.1-intel.0"
```

Comment:
Please change the tag to "1.17.1-intel.0"

Reply:
okay, done!